README.md

```diff
@@ -73,27 +87,17 @@ The simulator has its own configuration,
 you can refer to the [documentation](./simulator/docs/simulator-server-config.md).
 
 See the following docs to know more about simulator:
+- [existing-cluster-importing.md](simulator/docs/existing-cluster-importing.md): describes how you can import resources in your cluster to the simulator so that you can simulate scheduling based on your cluster's situation.
```

Comment:
In #300, this feature was renamed.

And so sorry, I have given a similar correction in this #302. I'd like to call its feature like `Importing cluster's resource`. WDYT?
https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/302/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R163